packages/semantic-ui/src/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -41,7 +41,7 @@ export default function CheckboxWidget<
     formContext,
     uiSchema,
     defaultSchemaProps: {
-      inverted: false,
+      inverted: 'false',
```

Comment:
not really familiar with this, why is this a string and not boolean?

Reply:
There was an error in the console complaining about providing a boolean for a string property, so I fixed it